### DISCUSSION TITLE
Not initialized 问题

### DISCUSSION THREAD

**Discussion by obhcy** [walinejs/waline#1161] [Q&A] [answered]
按vercel部署文档完成部署后出现 Not initialized
![image](https://user-images.githubusercontent.com/34027743/176581609-41833062-b9d2-4492-8816-7d2a0204f326.png)

请问如何解决？感谢🙇‍

**Comment by lizheming** [comment, marked as answer]
https://github.com/walinejs/waline/issues?q=is%3Aissue+Not+initialized+is%3Aclosed
